pkg/cachemanager/cachemanager_test/cachemanager_integration_test.go

```diff
@@ -329,5 +417,6 @@ func cacheManagerForTest(t *testing.T, mgr manager.Manager, wm *watch.Manager, r
 		cancelFunc()
 		ctx.Done()
 	})
-	return cacheManager, cfClient, aggregator, ctx
+
+	return testResources{cacheManager, cfClient, aggregator}, ctx
```

Comment:
Should the `context` be separate from the rest of the struct?

Reply:
Going to resolve this for now but feel free to open for any follow ups.